xblockutils/public/studio_container.js

```diff
@@ -1,20 +1,21 @@
 function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
     var $buttons = $(".add-xblock-component-button", element),
+        $view = $('.add-xblock-component', element),
```

Comment:
@eliangcs I'm wondering if there is a better (more appropriate) name that you could use for this than `$view`: xblock-utils doesn't use Backbone, so the "view" terminology doesn't really apply.